app/routes/admin/messages/list.js

```diff
@@ -1,7 +1,18 @@
 import Route from '@ember/routing/route';
+import EmberTableRouteMixin from 'open-event-frontend/mixins/ember-table-route';
 
-export default Route.extend({
-  model() {
-    return this.modelFor('admin.messages');
+export default class extends Route.extend(EmberTableRouteMixin) {
+
+  async model(params) {
```

Comment:
@kushthedude is async really necessary here?

Reply:
I don't think it's really necessary but I won't like to break the regular routes defined for admin route